superset-frontend/src/dashboard/components/menu/HoverMenu.tsx

```diff
@@ -70,6 +72,18 @@ export default class HoverMenu extends React.PureComponent<HoverMenuProps> {
     children: null,
   };
 
+  handleMouseEnter = () => {
+    if (this.props.onHover) {
+      this.props.onHover(true);
+    }
+  };
+
+  handleMouseLeave = () => {
+    if (this.props.onHover) {
+      this.props.onHover(false);
```

Comment:
This looks great. Another small nit, I'd suggest being more explicit when defining this onHover function above. When calling it here, there's no context about what this boolean means. If possible, I usually like to pass a dictionary instead of a boolean argument, so `this.props.onHover({isHovered:false})` for example.

Reply:
Thank you for this suggestion by the way, details like this really help me get better!